### DISCUSSION TITLE
Link should open in Mobile but doesn't

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#117633] [Mobile]
First step, on your cell phone visit https://play.google.com/store/apps/details?id=org.quantumbadger.redreader

Second step, scroll down to where it says App support and click the Website link. 

Yes you'll end up on GitHub, but you'll end up in a browser. 

Okay now examining this process again, you'll see there's an intermediary redirect link, http://redreader.org , and if you copy and visit that then indeed you end up in getting to the final destination in your GitHub app.

**Comment by eliperkins** [comment]
Hey @jidanni , thanks for this step-by-step. From my understanding, it seems like `http://redreader.org/` is a URL that redirects to https://github.com/QuantumBadger/RedReader

```
~
❯ curl -v https://redreader.org
> Host: redreader.org
> User-Agent: curl/8.6.0
> Accept: */*
>
< HTTP/1.1 302 Found
< Date: Mon, 08 Apr 2024 17:20:36 GMT
< Server: Apache
< Location: https://github.com/QuantumBadger/RedReader
< Content-Length: 1
< Content-Type: text/html; charset=UTF-8
<

* Connection #0 to host redreader.org left intact
```

For my understanding, is the expectation here that the redirect should open _into_ GitHub Mobile, but instead opens in a browser?

**Reply by jidanni** [reply]
Yes. And what's worse, we end up with in the browser. But there's no way to then get into the app at the right spot, without a whole lot of manual handstands.
